examples/pytorch/text-generator/README.md

```diff
@@ -106,18 +102,6 @@ $ cortex logs text-generator
 ...
 ```
 
-Once your API is live, use `curl` to test your API (it will take a few seconds to generate the text):
-
-```bash
-$ curl http://localhost:8889 \
-    -X POST -H "Content-Type: application/json" \
-    -d '{"text": "machine learning is"}'
-
-"machine learning is ..."
-```
-
-<br>
-
 ## Deploy your model to AWS
```

Comment:
Since we are now thinking of Cortex in a more cluster-oriented fashion, should we make this section the first one to come? I'd push the `Deploy your model locally` section at the end. What's your take? We can talk about this live.